### PR TITLE
Add Django User Management files

### PR DESCRIPTION
**Pull request by acsany**
Since this is a tutorial update, there are no `step/` folders

### CODE REVIEW

File: django-user-management/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```bash
+(venv) $ python -m pip install -r requirements.txt
+```
+
+Make and apply the migrations for the project to build your local database:
+
+```bash
+(venv) $ python manage.py makemigrations
+(venv) $ python manage.py migrate
+```
```
Comment (bzaczynski) [inline django-user-management/README.md:32]: There's a missing step:

```
$ cd user_auth_intro/
```

File: django-user-management/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Django==5.1.3
```
Comment (bzaczynski) [inline django-user-management/requirements.txt:1]: We should probably include all requirements for reproducibility:

```
asgiref==3.8.1
Django==5.1.3
sqlparse==0.5.2
```

**Review by bzaczynski — CHANGES_REQUESTED** [review]
@acsany Overall, this looks good. I only left two small comments as of now, but let's wait until we sort out the tutorial first.

**Review by bzaczynski — APPROVED** [review]
@acsany LGTM!

**Review by brendaweles — APPROVED** [review]

**Pull request merged**